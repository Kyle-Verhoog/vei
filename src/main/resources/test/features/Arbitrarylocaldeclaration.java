public class Arbitrarylocaldeclaration {
  public Arbitrarylocaldeclaration() {}
  public void m() {
    int x = 42;
    x = x+1;
    int y = x+87;
  }
}