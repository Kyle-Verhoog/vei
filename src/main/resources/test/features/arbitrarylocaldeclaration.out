PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
VOID ┃ void
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
INT ┃ int
IDENTIFIER ┃ x
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
IDENTIFIER ┃ x
= ┃ =
IDENTIFIER ┃ x
+ ┃ +
INTEGER_LITERAL ┃ 1
; ┃ ;
INT ┃ int
IDENTIFIER ┃ y
= ┃ =
IDENTIFIER ┃ x
+ ┃ +
INTEGER_LITERAL ┃ 87
; ┃ ;
} ┃ }
} ┃ }